### ISSUE TITLE
Installation cannot be completed as requested with CDT 9.10  (IEP-198)

### ISSUE THREAD

**Issue by rherrmannr** [espressif/idf-eclipse-plugin#93]
**Describe the bug**
I'm trying to install Espressif into a clean Eclipse CDT 2019-12 using CDT 9.10.

**To Reproduce**
1. Download Eclipse C/C++ 2019-12 here:
https://www.eclipse.org/downloads/packages/release/2019-12/r
2. unzip, run
3. install new software as described here:
https://github.com/espressif/idf-eclipse-plugin/blob/master/README.md
4. Following error:
```
Cannot complete the install because of a conflicting dependency.
  Software being installed: Espressif IDF Plugins for Eclipse 1.1.1.202006040612 (com.espressif.idf.feature.feature.group 1.1.1.202006040612)
  Software currently installed: C/C++ Qt Support - Preview 9.10.0.201911192140 (org.eclipse.cdt.qt.feature.group 9.10.0.201911192140)
  Only one of the following can be installed at once: 
    Template Engine Freemarker Support (Java 11) 1.1.100.202001140305 (org.eclipse.tools.templates.freemarker.java11 1.1.100.202001140305)
    Template Engine Freemarker Support (Java 11) 1.1.2.201911192140 (org.eclipse.tools.templates.freemarker.java11 1.1.2.201911192140)
  Cannot satisfy dependency:
    From: Espressif IDF Plugins for Eclipse 1.1.1.202006040612 (com.espressif.idf.feature.feature.group 1.1.1.202006040612)
    To: org.eclipse.equinox.p2.iu; org.eclipse.tools.templates.freemarker.java11 [1.1.100.202001140305,1.1.100.202001140305]
  Cannot satisfy dependency:
    From: C/C++ Qt Support - Preview 9.10.0.201911192140 (org.eclipse.cdt.qt.feature.group 9.10.0.201911192140)
    To: org.eclipse.equinox.p2.iu; org.eclipse.tools.templates.freemarker.java11 [1.1.2.201911192140,1.1.2.201911192140]
```

**Expected behavior**
[Installing Prerequisites](https://dl.espressif.com/dl/idf-eclipse-plugin/updates/latest/) says Eclipse 2018-12 CDT and above are supported.

**Environment Information:** 

[config.txt](https://github.com/espressif/idf-eclipse-plugin/files/4729812/config.txt)

**Comment by kolipakakondal** [comment]
Thanks for reporting. I'm looking into it.

**Comment by kolipakakondal** [comment]
Hello @rherrmannr 

Until we provide a fix for this issue, you can choose one of these options to proceed with the installation.
1. **Upgrade to Eclipse CDT 2020-03**  - If you've already had Eclipse CDT 2020-03 you don't get into this issue.
2. Choose "**Update my installation to compatible with the items being installed**" and proceed with the installation. This basically will update your Eclipse CDT version to 2020-03

<img width="920" alt="Screenshot 2020-06-05 at 3 58 32 PM" src="https://user-images.githubusercontent.com/8463287/83867032-5b53f680-a746-11ea-8558-e6e7a70b2a24.png">

**Comment by rherrmannr** [comment]
Hey @kolipakakondal 

Thanks for your help. I've already seen this workaround, but this won't work for me, because of other dependencies in my eclipse that force me to use CDT 9.10.

Seems like I must wait for your fix or somehow update to CDT 9.11.

**Comment by thehellmaker** [comment]
Has the same issue on 2020-06 version of eclipse. Downgraded to 2020-03 it was resolved. Just as FYI

**Comment by kolipakakondal** [comment]
We recommend using Java 11 and above while working with IDF Eclipse Pluginconsidering Eclipse 2020-06 has a requirement for Java 11 to work with the CDT

I've added a note on this here https://github.com/espressif/idf-eclipse-plugin/releases/tag/v1.1.1

**Comment by groskopf** [comment]
I had similar problems. I just upgraded my openJdk jre from version 8 to 14. Then it worked.

**Comment by rherrmannr** [comment]
I did not test this so far but perhaps this works for others. You can skip the JRE 11 compatibility check:
https://www.eclipse.org/lists/cdt-dev/msg34492.html

**Comment by kolipakakondal** [comment]
Hi All,

We have released ESP-IDF Eclipse Plugin v1.2.1 (Build: 1.2.1.202008251732). Please check this here https://github.com/espressif/idf-eclipse-plugin/releases/tag/v1.2.1

Use Java 11 and above, Eclipse 2020-06(4.16) to work with the IDF Eclipse plugin

Thanks
